Makefile

```diff
@@ -7,8 +7,8 @@ WMCO_VERSION ?= 10.17.1
 
 # *_GIT_VERSION are the k8s versions. Any update to the build line could potentially require an update to the sed
 # command in generate_k8s_version_commit() in hack/update_submodules.sh
-KUBELET_GIT_VERSION=v1.30.6+5a24e85
-KUBE-PROXY_GIT_VERSION=v1.30.0
+KUBELET_GIT_VERSION=v1.30.7+f868289
+KUBE-PROXY_GIT_VERSION=v1.31.1
```

Comment:
KUBE-PROX should be aligned with KUBELET. 1.31 is for release-4.18 for example: 

https://github.com/openshift/windows-machine-config-operator/pull/2627/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R10